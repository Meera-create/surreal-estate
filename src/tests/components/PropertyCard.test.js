import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import PropertyCard from '../../components/propertyCard';


describe('testing props', () => {
    test('has the correct value been asserted for title prop', () => {
        const { asFragment } = render(<PropertyCard title />);
        expect(asFragment()).toMatchSnapshot();
    })
   
})

