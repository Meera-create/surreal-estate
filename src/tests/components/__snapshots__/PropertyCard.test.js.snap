// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing props has the correct value been asserted for title prop 1`] = `
<DocumentFragment>
  <div
    class="property_lists"
  >
    <p />
    <ul>
      <li />
    </ul>
    <p />
  </div>
</DocumentFragment>
`;
